### PR TITLE
🌐 Add Korean translation for `docs/ko/docs/advanced/websockets.md`

### PR DESCRIPTION
**Pull request by kwang1215**
Add translation to docs/ko/docs/advanced/websockets.md

### CODE REVIEW

File: docs/ko/docs/advanced/websockets.md [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+
+## WebSockets 클라이언트
+
+### 프로덕션 환경에서
+
+여러분의 프로덕션 시스템에서는 React, Vue.js 또는 Angular와 같은 최신 프레임워크로 생성된 프런트엔드를 사용하고 있을 가능성이 높습니다.
+
+백엔드와 WebSockets을 사용해 통신하려면 아마도 프런트엔드의 유틸리티를 사용할 것입니다.
+
+또는 네이티브 코드로 WebSocket 백엔드와 직접 통신하는 네이티브 모바일 애플리케이션을 가질 수도 있습니다.
```
Comment (Limsunoh) [inline docs/ko/docs/advanced/websockets.md:27]:
```suggestion
또는 네이티브 코드로 WebSocket 백엔드와 직접 통신하는 네이티브 모바일 응용 프로그램을 가질 수도 있습니다.
```

'애플리케이션' should be changed to '응용 프로그램'.

File: docs/ko/docs/advanced/websockets.md [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+
+프로덕션 환경에서는 위에서 설명한 옵션 중 하나를 사용하는 것이 좋습니다.
+
+그러나 이는 WebSockets의 서버 측에 집중하고 동작하는 예제를 제공하는 가장 간단한 방법입니다:
+
+{* ../../docs_src/websockets/tutorial001.py hl[2,6:38,41:43] *}
+
+## `websocket` 생성하기
+
+**FastAPI** 애플리케이션에서 `websocket`을 생성합니다:
```
Comment (Limsunoh) [inline docs/ko/docs/advanced/websockets.md:45]:
```suggestion
**FastAPI** 응용 프로그램에서 `websocket`을 생성합니다:
```

'애플리케이션' should be changed to '응용 프로그램'.

File: docs/ko/docs/advanced/websockets.md [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+
+간단한 페이지가 나타날 것입니다:
+
+<img src="/img/tutorial/websockets/image01.png">
+
+입력창에 메시지를 입력하고 전송할 수 있습니다:
+
+<img src="/img/tutorial/websockets/image02.png">
+
+**FastAPI** WebSocket 애플리케이션이 응답을 돌려줄 것입니다:
```
Comment (Limsunoh) [inline docs/ko/docs/advanced/websockets.md:89]:
```suggestion
**FastAPI** WebSocket 응용 프로그램이 응답을 돌려줄 것입니다:
```

'애플리케이션' should be changed to '응용 프로그램'.

File: docs/ko/docs/advanced/websockets.md [hunk 4]
```diff
@@ -0,0 +1,186 @@
...
+
+WebSocket에서는 `HTTPException`을 발생시키는 것이 적합하지 않습니다. 대신 `WebSocketException`을 발생시킵니다.
+
+명세서에 정의된 <a href="https://tools.ietf.org/html/rfc6455#section-7.4.1" class="external-link" target="_blank">유효한 코드</a>를 사용하여 종료 코드를 설정할 수 있습니다.
+
+///
+
+### 종속성을 가진 WebSockets 테스트
+
+파일 이름이 `main.py`라고 가정하고 애플리케이션을 실행합니다:
```
Comment (Limsunoh) [inline docs/ko/docs/advanced/websockets.md:124]:
```suggestion
파일 이름이 `main.py`라고 가정하고 응용 프로그램을 실행합니다:
```

'애플리케이션' should be changed to '응용 프로그램'.

File: docs/ko/docs/advanced/websockets.md [hunk 5]
```diff
@@ -0,0 +1,186 @@
...
+
+`WebSocketDisconnect` 예외가 발생하며, 다른 모든 클라이언트가 다음과 같은 메시지를 수신합니다:
+
+```
+Client #1596980209979 left the chat
+```
+
+/// tip | 팁
+
+위 애플리케이션은 여러 WebSocket 연결에 메시지를 브로드캐스트하는 방법을 보여주는 간단한 예제입니다.
```
Comment (Limsunoh) [inline docs/ko/docs/advanced/websockets.md:173]:
```suggestion
위 응용 프로그램은 여러 WebSocket 연결에 메시지를 브로드캐스트하는 방법을 보여주는 간단한 예제입니다.
```

'애플리케이션' should be changed to '응용 프로그램'.

Comment (kwang1215) [reply docs/ko/docs/advanced/websockets.md]: Thank you for your reviews !!

**Review by Limsunoh — COMMENTED** [review]
LGTM 👍

**Review by kim-sangah — APPROVED** [review]
LGTM!

**Comment by alejsdev** [comment]
Thanks for your contribution! @kwang1215 :rocket: 
And thanks for your review @Limsunoh :nerd_face:

**Pull request merged**